martian-filetypes/Cargo.toml

```diff
@@ -14,7 +14,7 @@ martian-derive = { path = "../martian-derive" }
 serde = { version = '1.0', features = ['derive'] }
 serde_json = "*"
 bincode = "1.3"
-failure = "*"
+anyhow = { version = "1", features = ["backtrace"] }
```

Comment:
Not sure how I feel about enabling this feature by default.  I might hide it behind a feature flag.

Really I don't think it's terribly useful to enable the feature in this crate at all.  It's needed in the `martian` crate to support stackvars.  Nearly all users of this crate will also be using `martian`, which means they'll get that feature enabled anyway, but if they aren't using `martian` then forcing them to enable it here seems unnecessary.